src/logmessage.cpp

```diff
@@ -27,35 +27,35 @@ namespace g3 {
    // helper for setting the normal log details in an entry
    std::string LogDetailsToString(const LogMessage& msg) {
       std::string out;
-      out.append("\n" + msg.timestamp() + "\t"
-                 + msg.level() + " [" + msg.file() + "->" + msg.function() + ":" + msg.line() + "]\t");
```

Comment:
@rickyzhang82 I believe @cstamatopoulos is referring to the alignment in the text log. I.e. the tab alignment  is missing in the PR

![screen shot 2016-12-22 at 12 53 26 pm](https://cloud.githubusercontent.com/assets/331742/21438358/b3314526-c845-11e6-99a8-c0037a60aa4e.png)

Could you put it back again please?